<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initial_conditions</class>
 <widget class="QMainWindow" name="initial_conditions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Початково-крайовий стан</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Початково-крайові умови</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Кількість умов (R_0)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Кількість точок дискретизації (L_0)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Кількість похідних по t</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="condNum">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="dotsNum">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="derNum">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ic.qrc"/>
 </resources>
 <connections/>
</ui>
